### PR TITLE
Update doc to include [ccpp-table-properties]:

### PR DESCRIPTION
**Pull request by JulieSchramm**
- Update CCPPPreBuild.rst and _static/ccpp_prebuild_static_only.png to remove
    SCHEME_FILES_DEPENDENCIES and update example of ccpp_prebuild_config.py for SCM

  - Update CompliantPhysicsParams.rst to include [ccpp-table-properties] and
    update listings to include this new table section

  - Remove CCPP=N from ConfigBuildOptions.rst

  - Update HostSideCoding.rst to include [ccpp-table-properties]

  - Update _static/scheme_template.meta to include [ccpp-table-properties]

### CODE REVIEW

**Review by climbfuji — COMMENTED** [review]
Looks good to me, but I would like to see `CCPPtechnical/source/ConfigBuildOptions.rst` in its final form, if possible. Thank you!

**Comment by JulieSchramm** [comment]
Here is the PDF file. See Chapter 3.  

[CCPPtechnical.pdf](https://github.com/NCAR/ccpp-doc/files/5387294/CCPPtechnical.pdf)

**Comment by climbfuji** [comment]
> Here is the PDF file. See Chapter 3.
> 
> [CCPPtechnical.pdf](https://github.com/NCAR/ccpp-doc/files/5387294/CCPPtechnical.pdf)

Thanks. It looks good to me. In the third paragraph of chapter three, we speak about `GFS_physics_driver.F90` etc in a context that includes both SCM and the UFS. These files, however, only exist (yes, they still do) in the UFS, not in the SCM. But maybe we don't need to be that picky.

**Review by climbfuji — APPROVED** [review]

**Comment by JulieSchramm** [comment]
I saw this sentence and wasn't sure if these files were still used.  They
still exist in the UFS model.  Do you have a suggestion on how to edit the
wording?

**Comment by climbfuji** [comment]
Leave it for now, is my suggestion. They will be gone in a month time, and then we have to rewrite the paragraph anyway.

File: CCPPtechnical/source/ConfigBuildOptions.rst [hunk 1]
```diff
@@ -9,11 +9,7 @@ The SCM and the UFS Atmosphere are supported for use with the CCPP. In the case
...
+By default, the code is compiled with CCPP enabled and restricted to CCPP-compliant physics. That is, any parameterization to be called as part of a suite must be available in CCPP. Physics scheme selection and order is determined at runtime by an external suite definition file (SDF; see :numref:`Chapter %s <ConstructingSuite>` for further details on the SDF). The existing physics-calling code ``GFS_physics_driver.F90`` and ``GFS_radiation_driver.F90`` are bypassed altogether in this mode and any additional code needed to connect parameterizations within a suite previously contained therein is executed from the so-called CCPP-compliant “interstitial schemes”.  Compiling with CCPP links *CCPP-Framework* and *CCPP-physics* to the executable and is restricted to CCPP-compliant physics defined by one or more SDFs used at compile time.
```
Comment (ligiabernardet) [inline CCPPtechnical/source/ConfigBuildOptions.rst:12]: I don't think we should talk about ``GFS_physics_driver.F90`` and ``GFS_radiation_driver.F90``as many people will not know what this is. This is historical information.
```suggestion
By default, the code is compiled with CCPP enabled, that is, *CCPP-Framework* and *CCPP-physics* are linked to the executable. Any parameterization to be called as part of a suite must be CCPP-compliant and available in *CCPP-Physics*.  Physics scheme selection and order is determined at runtime by an external suite definition file (SDF; see :numref:`Chapter %s <ConstructingSuite>` for further details on the SDF). 
```

File: CCPPtechnical/source/HostSideCoding.rst [hunk 2]
```diff
@@ -25,9 +25,7 @@ At present, only two types of variable definitions are supported by the CCPP-Fra
 Metadata for Variable in the Host Model
 ==================================================
 
-To establish the link between host model variables and physics scheme variables, the host model must provide metadata information similar to those presented in :numref:`Section %s <MetadataRules>`. The host model can have multiple metadata files (``.meta``) with multiple metadata sections in each file 
-(``[ccpp-arg-table]``) or just one. The host model Fortran files contain three-line snippets to indicate the location for insertion of the 
-metadata information contained in the corresponding section in the ``.meta`` file.
+To establish the link between host model variables and physics scheme variables, the host model must provide metadata information similar to those presented in :numref:`Section %s <MetadataRules>`. The host model can have multiple metadata files (``.meta``), each with the required ``[ccpp-table-properties]`` section and the related ``[ccpp-arg-table]`` sections in each file. The host model Fortran files contain three-line snippets to indicate the location for insertion of the metadata information contained in the corresponding section in the ``.meta`` file.
```
Comment (ligiabernardet) [inline CCPPtechnical/source/HostSideCoding.rst:28]: Word each appears twice in a sentence
```suggestion
To establish the link between host model variables and physics scheme variables, the host model must provide metadata information similar to those presented in :numref:`Section %s <MetadataRules>`. The host model can have multiple metadata files (``.meta``), each with the required ``[ccpp-table-properties]`` section and the related ``[ccpp-arg-table]`` sections. The host model Fortran files contain three-line snippets to indicate the location for insertion of the metadata information contained in the corresponding section in the ``.meta`` file.
```

**Review by ligiabernardet — CHANGES_REQUESTED** [review]
Thanks for the documentation.

Unrelated to the modifications here: I noticed that the PDF (Page 2) is telling viewers to reference this document by citing v4.0. But this PDF does not correspond to any particular version. I think we should remove the citation altogether from the master code (and only have it in documentation associated with a release)

**Comment by JulieSchramm** [comment]
Can this be merged?  I resolved the two paragraphs using the GitHub page. I'm not sure about what to do about the links on page 2.  I don't want the links to create a merge conflict every time we have to merge a release branch back to master.

**Review by ligiabernardet — APPROVED** [review]
In this case the citation in the master branch will have to always refer to the latest release.

**Pull request merged**